### ISSUE TITLE
Сan not activate the Commented Rules

### ISSUE THREAD

**Issue by esmelnikov** [StamusNetworks/SELKS#85]
Hello!
Firstly, thank you so much for your work!
I installed SELKS-4.0rc1-desktop.iso. After installing the setup, everything worked fine, except for the problem described by jasonish (the problems helped to solve the advice from the branch "SELKS 4: IP address not obtained on boot.").
After the system was updated with the "/opt/selks/Scripts/Setup/selks-upgrade_stamus.sh" script, the following problems with the Scirius web interface appeared: you can not activate the Commented Rules, i.e. After clicking on the link Action -> Toggle availability rule is not activated.

**Comment by pevma** [comment]
Thanks for trying it out - the stable release is planned for this week :)

The problem with the IP DHCP is solved by installing - the net-tools package.

Which Scirius version is that ? 1.2.4?
How do you know the rules are not activated - I think you need to update and push/restart for those to take effect - were you able to do that ?

**Comment by esmelnikov** [comment]
I'm using Scirius version 1.2.4.
After clicking on the link, the state of the rule does not change, i.e. No additional menu items are displayed in the "Action" section.

**Comment by pevma** [comment]
Did you try to update and push the ruleset? (from the Suricata tab in Scirius)

**Comment by esmelnikov** [comment]
I tried to update the script: /opt/selks/Scripts/Setup/selks-upgrade_stamus.sh, but received the following errors:

```
W: GPG error: http://ppa.launchpad.net/webupd8team/java/ubuntu precise InRelease: The following signatures could not be verified because the public key is not available: NO_PUBKEY C2518248EEA14886
W: The repository 'http://ppa.launchpad.net/webupd8team/java/ubuntu precise InRelease' is not signed.
N: Data from such a repository can not be authenticated and is therefore appropriately dangerous to use.
N: See apt-secure (8) manpage for repository creation and user configuration details.
```
How can I fix this problem?

**Comment by pevma** [comment]
It is a warning  -  from the Oracle deb package repository. You can continue forward with the update.

**Comment by esmelnikov** [comment]
The problem persisted, it was found out beforehand that the disabling of the rules: "Action -> Disable handle" also does not work.

**Comment by pevma** [comment]
It seems the "Toggle availability" is not working for me either.
I can disable a rule though - no problem - it changes the status form `Active` to `Inactive`

**Comment by esmelnikov** [comment]
Sorry. I confirm that the rules are turned off, I was confused by the lack of changes in the menu "Action" by analogy with the option "Toggle availability"

**Comment by pevma** [comment]
Ok -  thanks for checking.
So -  we still do have the `Toggle availability` issue correct? If so would you please post a bug report on the Scirius Github page?

**Comment by esmelnikov** [comment]
Yes, issue correct. I posted a bug report  [#114](https://github.com/StamusNetworks/scirius/issues/114) on the Scirius Github page.
